sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultCertificatesInfo.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import java.security.Key;
+import java.security.cert.Certificate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Date;
+import java.util.Collections;
+import java.util.Objects;
+
+/**
+ * Store certificates in portal
+ */
+public class KeyVaultCertificatesInfo {
+
+    /**
+     * Stores the list of aliases.
+     */
+    private List<String> aliases;
+
+    /**
+     * Stores the certificates by alias.
+     */
+    private final Map<String, Certificate> certificates = new HashMap<>();
+
+    /**
+     * Stores the certificate keys by alias.
+     */
+    private final HashMap<String, Key> certificateKeys = new HashMap<>();
+
+    /**
+     * Stores the last time refresh certificates and alias
+     */
+    private Date lastRefreshTime;
+
+    private final long refreshInterval = Optional.ofNullable(System.getProperty("azure.keyvault.jca.certificates-refresh-interval"))
+        .map(Long::valueOf)
+        .orElse(0L);
+
+    boolean certificatesNeedRefresh() {
+        if (refreshInterval > 0) {
+            return lastRefreshTime.getTime() + refreshInterval < new Date().getTime();
+        }
+        return false;
+    }
+
+    List<String> getAliases(KeyVaultClient keyVaultClient) {
+        if (lastRefreshTime == null || certificatesNeedRefresh()) {
+            getLatestCertificates(keyVaultClient);
+        }
+        return aliases;
+    }
+
+    Map<String, Certificate> getCertificates(KeyVaultClient keyVaultClient) {
+        if (lastRefreshTime == null || certificatesNeedRefresh()) {
+            getLatestCertificates(keyVaultClient);
+        }
+        return certificates;
+    }
+
+    Map<String, Key> getCertificateKeys(KeyVaultClient keyVaultClient) {
+        if (lastRefreshTime == null || certificatesNeedRefresh()) {
+            getLatestCertificates(keyVaultClient);
+        }
+        return certificateKeys;
+    }
+
+    void getLatestCertificates(KeyVaultClient keyVaultClient) {
```

Comment:
1. if return type is void, I don't think getLatestCertificates is a good name here.
2. shoud we add "private" or "public" here?